### PR TITLE
redirect_addr property added to consul backend

### PR DESCRIPTION
**Pull request by paweloczadly**
We need redirect_addr property for the Consul backend. Could you review and merge if the PR is ok?

### CODE REVIEW

File: jobs/vault/templates/bin/vault_ctl [hunk 1]
```diff
@@ -18,7 +18,8 @@ case $1 in
     echo $$ > $PIDFILE
 
     exec vault server -config=$JOB_DIR/config/server.hcl \
-         >>$LOG_DIR/$JOB_NAME.log
+         >>$LOG_DIR/$JOB_NAME.log \
+         2>>$LOG_DIR/$JOB_NAME.stderr.log
```
Comment (jhunt) [inline jobs/vault/templates/bin/vault_ctl]: Please remove this change, it is unrelated.

**Review by jhunt — CHANGES_REQUESTED** [review]

File: jobs/vault/templates/bin/vault_ctl (hunk 1 above)
Comment (paweloczadly) [reply jobs/vault/templates/bin/vault_ctl]: should I squash all commits and create a new PR?

Comment (jhunt) [reply jobs/vault/templates/bin/vault_ctl]: Nah, I'll handle that via Github when I merge the PR.

Comment (paweloczadly) [reply jobs/vault/templates/bin/vault_ctl]: perfect 👍

**Pull request merged**